### DISCUSSION TITLE
Heroku

### DISCUSSION THREAD

**Discussion by rohitdeka-1** [community/community#147248] [GitHub Education] [answered]
### Hi everyone,

I used github pack to get credits on heroku , I added my card did the 1$ payment also received the credits from heroku can I now remove card and the e mandate that heroku made on my bank ?

**Comment by davevad93** [comment, marked as answer]
Hi @rohitdeka-1 ,

To avoid problems, I suggest you to leave your billing information on `Heroku`, as you will receive 13$ of credits every month for a total of 24 months.

**Reply by rohitdeka-1** [reply]
But I have already got the whole 322$ credit

**Reply by MohamedYassineKhaldi00** [reply]
@rohitdeka-1 can you give me updates ? Have you removed the card ?

**Reply by rohitdeka-1** [reply]
No I haven't @MohamedYassineKhaldi00
